### DISCUSSION TITLE
End of Mainstream / General Support for Operating System Examples

### DISCUSSION THREAD

**Discussion by tenthirtyam** [vmware/packer-examples-for-vsphere#855] [Polls]
The project generally drops guest operating systems from this project after end of general or mainstream support. 

For example, over time the following have been removed:

- Windows Server 2016
- Ubuntu 18.04 LTS

The project will likely drop Ubunto 23.10 when 24.04 LTS releases may be dropping some of the CentOS 7, Stream 8, and RHEL7 later this year.

**Comment by Kennochas** [comment]
If the project supports a platform: IE Ubuntu, if its a version that's under active support / customers can use it, it should probably be considered as supported of the project.

We are a VMWare Cloud Foundations (VCF) Pinnacle Partner, and use this along with William Lam's python script to create templates for our customers / automation to consume from a S3 backed content library.

The situation that brought this up: Server 2016, is under active support until 2027 from Microsoft. Microsoft partners are still covered to install license and use it, as well as customers purchase. 

The major difference at-least with Ubuntu 18.04, is it isn't getting security updates because it's completely out of support except if you use Ubuntu Pro and pay for it. 

Server 2016 is getting full security updates, and listed as a current OS: https://learn.microsoft.com/en-us/windows/release-health/windows-server-release-info still, because its still actively receiving security bug-fixes.

**Reply by kmcgrego1307** [reply]
If Windows 2016 is still actively getting patches from Microsoft (if you're paying for it), and if Ubuntu 18.04 LTS is still actively getting patches from Canonical (if you're paying for it), then it feels like this project should still support them.

**Reply by Kennochas** [reply]
My point here is 2016 is getting active security updates and no payment is required. It’s in extended support mode. It’s essentially the equivalent of LTS support. It’s actively getting security updates but no new features.
